packages/dx-react-chart/src/plugins/area-series.jsx

```diff
@@ -1,14 +1,11 @@
 import {
-  dArea as path,
-  getAreaPointTransformer as getPointTransformer,
-  createAreaHitTester as createHitTester,
+  getAreaPointTransformer as getPointTransformer, createAreaHitTester as createHitTester,
 } from '@devexpress/dx-chart-core';
 import { declareSeries } from '../utils';
 import { Area as Path } from '../templates/series/area';
 
 export const AreaSeries = declareSeries('AreaSeries', {
   components: { Path },
-  path,
   getPointTransformer,
   createHitTester,
```

Comment:
As the `path` property is no longer passes through the `declareSeries` HOC, it is should be remove in that function too.
              `<currentSeries.seriesComponent
                    ...
                    path={currentSeries.path}
                    ...
                  />`

Reply:
Done.